### ISSUE TITLE
Adding a new block-storage volume to existing instance reboot the instance and change its private IP address

### ISSUE THREAD

**Issue by ballinette** [scaleway/terraform-provider-scaleway#529]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

Terraform v0.13.0
+ provider.scaleway v1.16.0

### Affected Resource(s)

* scaleway_instance_server
* scaleway_instance_volume

### Terraform Configuration Files

* 1st step: create an empty instance without additional volume:

```hcl
resource "scaleway_instance_server" "myinstance" {
  name   = "myinstance"
  image = "debian-buster"
  type = "DEV1-S"
}

```

* 2nd step: attach a new volume to existing instance:

```hcl
resource "scaleway_instance_volume" "myvolume" {
    type       = "b_ssd"
    name   = "myvolume"
    size_in_gb = 10
}
resource "scaleway_instance_server" "myinstance" {
  name   = "myinstance"
  image = "debian-buster"
  type = "DEV1-S"
  additional_volume_ids = [ scaleway_instance_volume.myvolume.id ]
}

```

### Expected Behavior

When applying change, we should just have a new volume attached to the instance without any other modifications, as said by the plan:
```
 # module.nomad.scaleway_instance_server.myinstance will be updated in-place
  ~ resource "scaleway_instance_server" "myinstance" {
      ~ additional_volume_ids = [] -> (known after apply)
        boot_type             = "local"
        image                 = "debian-buster"
// (...)
        private_ip            = "10.69.108.103"
        root_volume {
            delete_on_termination = true
            size_in_gb            = 20
            volume_id             = "fr-par-1/513ce75c-2e53-4ea9-8eb4-32a2028f7e58"
        }
    }
  # module.nomad.scaleway_instance_volume.myvolume will be created
  + resource "scaleway_instance_volume" "myvolume" {
      + id              = (known after apply)
      + name            = "myvolume"
      + organization_id = (known after apply)
      + server_id       = (known after apply)
      + size_in_gb      = 10
      + type            = "b_ssd"
      + zone            = (known after apply)
    }

```

### Actual Behavior

Once the new volume is created, the instance is rebooted, take a few minutes to be modified, and once rebooted, the volume is correctly attached, but the instance's private IP has been modified. (for example 10.69.138.147 now vs. 10.69.108.103 before)

**Comment by vincentfretin** [comment]
I just had a similar issue with private ip that changed (some times, not always) when an instance changed (if you modify the tags for example). The problem is that I reference the private ip for a load balancer, so here I need to run `terraform apply` a second time to verify if the private ips changed, if yes, apply the changes for the load balancer.

extract of my config:
```hcl
resource "scaleway_instance_server" "app" {
  count = 2
  ...
}

resource "scaleway_lb_backend_beta" "backend01" {
  ...
  server_ips = scaleway_instance_server.app[*].private_ip
}
```

I don't think this can be solved in the provider. The provider don't know in advance that the private ip will change, so it can't say that the load balancer config will be changed. It can maybe detect during the execution that the private ip changed, but it shouldn't apply changes to the load balancer config that I didn't validate.

**Comment by remyleone** [comment]
Should be solved with #708
